gap/examples.gi

```diff
@@ -370,3 +370,66 @@ GeneralisedPetersenGraphCons);
 
 InstallMethod(GeneralisedPetersenGraph, "for integer, integer", [IsInt, IsInt],
 {n, k} -> GeneralisedPetersenGraphCons(IsImmutableDigraph, n, k));
+
+InstallMethod(HaarGraphCons,
+"for IsMutableDigraph and one integer",
+[IsMutableDigraph, IsInt],
```

Comment:
I would probably just change `IsInt` -> `IsPosInt` here and not check in the function

Reply:
And also change the string above if you do!